### PR TITLE
Ability to specify the Adapter configuration in UI

### PR DESCRIPTION
**Pull request by nitinmotgi**

### CODE REVIEW

File: cdap-ui/server/aggregator.js [hunk 1]
```diff
@@ -105,6 +105,36 @@ Aggregator.prototype.stopPollingAll = function() {
...
+    config = require('../templates/' + templateid + '/' + pluginid + '.json');
+    statusCode = 200;
+  } catch(e1) {
+   try {
+     // Some times there might a plugin that is common across multiple templates
+     // in which case, this is stored within the common directory. So, if the 
+     // template specific plugin check fails, then attempt to get it from common.
+     config = require('../templates/common/' + pluginid + '.json');  
+     statusCode = 200;
+   } catch (e2) { }
```
Comment (anwar6953) [inline cdap-ui/server/aggregator.js]: log if e2 and respond accordingly?

File: cdap-ui/app/features/etlapps/services/etl-create-factory.js [hunk 2]
```diff
@@ -33,6 +33,17 @@ angular.module(PKG.name + '.feature.etlapps')
         }.bind(this));
     }
 
+    ETLAppsApiFactory.prototype.fetchUIConfigurations = function(templateId, pluginId) {
+      this.dataSrc.config({
+        templateid: templateId,
+        pluginid: pluginId
+      })
+       .then(function(res) {
+         console.log(res);
```
Comment (anwar6953) [inline cdap-ui/app/features/etlapps/services/etl-create-factory.js]: plan to keep this here?

**Comment by ajainarayanan** [comment]
:+1: LGTM apart from @anwar6953's comments.

**Comment by anwar6953** [comment]
lgtm besides the two minor comments.

File: cdap-ui/server/aggregator.js (hunk 1 above)
Comment (nitinmotgi) [inline cdap-ui/server/aggregator.js]: done!

File: cdap-ui/app/features/etlapps/services/etl-create-factory.js (hunk 2 above)
Comment (nitinmotgi) [inline cdap-ui/app/features/etlapps/services/etl-create-factory.js]: yep!

**Pull request merged**